sample-config-advanced.json

```diff
@@ -519,23 +519,23 @@
           "cert": "saml.pem"
         },
         "oidc": {
-          "authorizationURL": "https://sso.server.com/api/oidc/authorization",
-          "callbackURL": "https://mesh.server.com/oidc-callback",
-          "clientid": "00000000-0000-0000-0000-000000000000",
-          "clientsecret": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-          "issuer": "https://sso.server.com",
-          "tokenURL": "https://sso.server.com/api/oidc/token",
-          "userInfoURL": "https://sso.server.com/api/oidc/userinfo",
-          "logoutURL": "https://sso.server.com/logout",
-          "newAccounts": true,
+          "issuer": {
+            "issuer": "https://sso.server.com",
+            "end_session_endpoint": "https://sso.server.com/logout"
+          },
+          "client": {
+            "clientid": "00000000-0000-0000-0000-000000000000",
```

Comment:
Should be `client_id`.

Reply:
This is how the node-openid-client module had it setup so I used those configs. This still accepts client_id and client_secret at the top level but if you put them under client you need to remove the underscore to be compliant with the schema I've setup.

Reply:
I had it backward and you are correct on both of these. I'll post a fix now.